scripts/setup-adapters.sh

```diff
@@ -62,7 +62,9 @@ function install_gcs-sdk-cpp {
 }
 
 function install_azure-storage-sdk-cpp {
-  github_checkout azure/azure-sdk-for-cpp azure-storage-blobs_12.8.0
+  vcpkg_commit_id=7a6f366cefd27210f6a8309aed10c31104436509
```

Comment:
Curious , would there be problems with previous installs of the azure storage library using this same script (i.e conflicts or picking up the wrong version etc)..